dash/scripts/circleci/collect-artifacts.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+[[ "$CIRCLECI" == "true" ]] || { echo "Not running on CircleCI"; exit 1; }
+
+mkdir -p /tmp/build-logs/${CIRCLE_JOB}
+mkdir -p /tmp/build-tests/${CIRCLE_JOB}
+
+cd build-ci
+
+echo "Collect Logs"
+cp -v --parents -R ./*/*/*/*.log /tmp/build-logs/${CIRCLE_JOB}
+
+echo "Collect Tests"
+cp -v --parents -R ./*/*/*/dash-tests-*.xml /tmp/build-tests/${CIRCLE_JOB}
+
+echo "Artifacts copied to /tmp/build-logs/${CIRCLE_JOB}"
+echo "Testlogs copied to /tmp/build-tests/${CIRCLE_JOB}"
```

Comment:
I suggest to move the output under the respective operation

Reply:
Yes, done!